### DISCUSSION TITLE
Split updates into one branch+PR for each chart that has changed?

### DISCUSSION THREAD

**Discussion by kastl-ars** [renovatebot/renovate#29208] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

currently CLI using 37.369.1

### Please tell us more about your question or problem

- Given a repo with multiple charts in chart/* directories.
- Given that some of them have the same outdated dependency.

In my tests I get one PR that changes that dependency in all of the Chart.yaml files and increases each chart's version.

Can I configure renovate to split this into separate branches and PRs for each chart?

I think I am missing the right search terms, because with "separate" and "branch" and "chart" and things like this I found nothing useful.

Thanks in advance!

Johannes

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
You can split them by parent directory by setting additionalBranchPrefix=parentDir

Search this repo's discussions as there are a few examples

**Reply by kastl-ars** [reply]
Thanks, now that I have something to look for I am sure I will find something.
